.github/workflows/ci.yml

```diff
@@ -41,9 +41,9 @@ jobs:
 
           # jruby-9.3
           - ruby: jruby-9.3
-            rails_version: '7.0.0'
+            rails_version: '6.0.0'
           - ruby: jruby-9.3
-            rails_version: 'edge'
+            rails_version: '6.1.0'
```

Comment:
Rails 7 and above requires Ruby 2.7 compatibility, however JRuby 9.3.x provides only Ruby 2.6 compatibility.

Test JRuby 9.3 in conjunction with Rails 6.0 and 6.1 instead.

Reply:
Can you flip this to jruby-9.4 instead please

Reply:
Updated in [`c2f29f3`](https://github.com/collectiveidea/delayed_job/compare/726f81fe7ac921f01a71d4e869fdb2c38bf8463a..c2f29f322d6bedf462084f2039d7a5e47f9b138d).